.eslintrc.json

```diff
@@ -2,6 +2,7 @@
   "extends": ["airbnb", "prettier", "plugin:node/recommended"],
   "plugins": ["prettier"],
   "rules": {
+    "max-lines": ["warn", 300, {"skipComments": true}],
```

Comment:
I really like this and it should stop us from creating monolithic files. Good call.